lib/opensusebasetest.pm

```diff
@@ -977,7 +977,7 @@ sub wait_boot_past_bootloader {
         # Workaround for bsc#1169723
         push(@tags, 'guest-disable-display');
     }
-    # bsc#1157928 - deal with additional polkit windows
+    # bsc#1177446 or bsc#1168979 - Polkit pop-up appears when it shouldn't
```

Comment:
Just a nipticking here: bsc#1168979 has no relation to bsc#1157928 or bsc#1177446

Reply:
Yes, now that I look at it, wait_boot_past_bootloader is called only in first_boot, so in first_boot should always be bsc#1177446, not bsc#1168979, and vice-versa (in shudown should always be bsc#1168979). I'll change this.